### PR TITLE
Organised dependencies

### PR DESCRIPTION
**Pull request by ukstv**
Instead of #845. No controversy, easy stuff here to silence lerna.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,24 +10,27 @@
     "publish:latest": "lerna publish --allow-branch master --conventional-graduate --create-release github",
     "lint": "lerna run lint",
     "clear-deps": "lerna exec 'rm -rf package-lock.json node_modules' ; rm -rf package-lock.json node_modules",
```
Comment (oed) [inline package.json:12]: I guess this line can be removed now that you added `clean:deps`.

File: packages/blockchain-utils-linking/src/cosmos.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { AccountID } from 'caip';
 import { AuthProvider } from './auth-provider';
 import { getConsentMessage, LinkProof } from './util';
-import { Tx, SignMeta } from '@tendermint/sig';
+import type { Tx, SignMeta } from '@tendermint/sig';
```
Comment (oed) [inline packages/blockchain-utils-linking/src/cosmos.ts:4]: Nice catch!

File: types/uint8arrays/index.d.ts [hunk 3]
```diff
@@ -3,3 +3,11 @@ declare module 'uint8arrays' {
...
     export function concat(bs: Array<Uint8Array>): Uint8Array
 }
+
+declare module 'uint8arrays/concat' {
+    export default function concat(bs: Array<Uint8Array>): Uint8Array
+}
+
+declare module 'uint8arrays/to-string' {
+    export default function toString(b: Uint8Array, enc?: string): string
+}
```
Comment (oed) [inline types/uint8arrays/index.d.ts:13]: Doesn't v 2.0.x include types?

**Review by oed — APPROVED** [review]

File: types/uint8arrays/index.d.ts (hunk 3 above)
Comment (ukstv) [reply types/uint8arrays/index.d.ts:13]: It does, sort of. https://github.com/achingbrain/uint8arrays/issues/13

Comment (oed) [reply types/uint8arrays/index.d.ts:13]: grr, ok thanks.

**Pull request merged**